WordPress/twentytwentyone
# Update the default pullquote color to gray

Fixes #837.

## Summary
Update the default pullquote color to gray. What is done is to add `color: var(--global--color-gray)` to the `wp-block-pullquote` class both in frontend and in editor files.


## Relevant technical choices:
No

## Test instructions

This PR can be tested by following these steps:

1. Add a pull quote block.
2. The color in the editor and in front should be gray (#39414d)
3. This color should also change in front and in editor when it's updated in the block settings

## Quality assurance
